cogs/help.py

```diff
@@ -3,28 +3,55 @@
-        super().__init__(timeout=None)
-    # Nextcord help
-    @ui.button(label="Nextcord help", style=ButtonStyle.red, custom_id="help:nextcord")
-    async def nextcord_help(self, _, interaction: Interaction):
-        await self.create_help_thread("Nextcord", interaction)
-    @ui.button(label="D.py libs help", style=ButtonStyle.gray, custom_id="help:dpylibs")
-    async def nextcord_lib_help(self, _, interaction: Interaction):
-        await self.create_help_thread("Discord.py libraries", interaction)
+help_topic = 'None'
+
+class Help_buttons(nextcord.ui.View):
     
-    # Python
-    @ui.button(label="Python help", style=ButtonStyle.green, custom_id="help:python")
-    async def python_help(self, _, interaction: Interaction):
-        await self.create_help_thread("Python", interaction)
-    async def create_help_thread(self, name, interaction):
-        thread = await interaction.channel.create_thread(name=f"{name} help ({interaction.user})", type=ChannelType.public_thread)
-        await interaction.response.send_message("Created!", ephemeral=True)
+    # Nextcord help butoon
+    @nextcord.ui.button(label='Nextcord Help?', style=nextcord.ButtonStyle.red)
+    async def Nextcord(self, button: nextcord.ui.Button, interaction: nextcord.Interaction):
+        view = Confirmation()
+        await interaction.response.send_message('Click "Yes!" to create the thread.', view=view, ephemeral=True)
+        global help_topic
+        help_topic = 'Nextcord'
+
+    # Discord.py help button
+    @nextcord.ui.button(label='Discord.py libs Help?', style=nextcord.ButtonStyle.blurple)
+    async def d_py(self, button: nextcord.ui.Button, interaction: nextcord.Interaction):
+        view = Confirmation()
+        await interaction.response.send_message('Click "Yes!" to create the thread.', view=view, ephemeral=True)
+        global help_topic
+        help_topic = 'D.py'
+      
+    # Python help button
+    @nextcord.ui.button(label='Python Help?', style=nextcord.ButtonStyle.green)
+    async def Python(self, button: nextcord.ui.Button, interaction: nextcord.Interaction):
+        view = Confirmation()
+        await interaction.response.send_message('Click "Yes!" to create the thread.', view=view, ephemeral=True)
+        global help_topic
+        help_topic = 'Python'
+        
+        
+class Confirmation(nextcord.ui.View):        
+        
+    @nextcord.ui.button(label='Yes!', style=nextcord.ButtonStyle.success)
+    async def Confirm(self, button: nextcord.ui.Button, interaction: nextcord.Interaction):
+        await interaction.response.send_message('Created the thread', ephemeral=True)
+        thread = await interaction.channel.create_thread(name=f" {help_topic} help ({interaction.user})", type=ChannelType.public_thread)
         await thread.add_user(interaction.user)
+        await thread.send(f'<@&882192899519954944> Help needed!\nAlright now that we are all here to help, what do you need help with {interaction.user.mention}?')
```

Comment:
It might be better if the mention is not hardcoded in the message.

Reply:
It was hardcoded in the original code also, i took these things from there to keep it same and simple